src/Microsoft.PowerShell.PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -323,8 +323,16 @@ internal static char GetCharFromConsoleKey(ConsoleKey key, ConsoleModifiers modi
 
             // get corresponding character  - may be 0, 1 or 2 in length (diacritics)
             var chars = new char[2];
-            int charCount = NativeMethods.ToUnicode(
-                virtualKey, scanCode, state, chars, chars.Length, NativeMethods.MENU_IS_INACTIVE);
+            int charCount = 1;
+            try
+            {
+                charCount = NativeMethods.ToUnicode(
```

Comment:
We should find a better way - this will cause an exception on every character typed, not good for performance and extremely annoying when debugging.

There is some one-time initialization code - so one less ugly option is to add a P/Invoke call there just to detect if the api is present or not, setting a flag that is checked here.